_glossary/local-transition.md

```diff
@@ -36,23 +36,25 @@ Local transitions are only possible to define _from_ a composite state and _to_
 
 ## xstate
 
-Xstate does not yet support local transitions.  It can be worked around by creating a single state that wraps all of the child states, and place the local transitions on the substate.
+In xstate, an internal transition is described by prefixing the target state with a dot.
 
 ```
-"ALPHABET": {
-  onentry: ...
-  onexit: ...
+{
+  initial: 'A',
+  onEntry: 'something',
+  onExit: 'something',
+  on: {
+    RESET: '.A'
```

Comment:
You can also specify an internal transition more explicitly:

```
RESET: [{ target: 'A', internal: true }]
```

Important to note that `internal: false` is the default (follows SCXML).

Reply:
Great, I'll make a PR updating the docs on Thursday.